eureka-webapp/src/main/java/edu/emory/cci/aiw/cvrg/eureka/servlet/worker/admin/SaveUserWorker.java

```diff
@@ -90,7 +90,7 @@ public void execute(HttpServletRequest req, HttpServletResponse resp)
 
 			this.servicesClient.updateUser(user,Long.valueOf(id));
 		} catch (ClientException e) {
-			throw new ServletException("Error saving user", e);
+			throw new ServletException("Error saving user's activate status and/or role", e);
```

Comment:
I believe the services call for saving the user could also return errors, so this exception is more general than just for errors in activation status and role.